src/test/resources/schema.sql

```diff
@@ -1,6 +1,17 @@
-drop table if exists champion;
-CREATE TABLE IF NOT EXISTS `champion` (
-    `id` integer primary key auto_increment,
-    `name` varchar(255) NOT NULL,
-    UNIQUE KEY `name` (`name`)
+--drop table if exists champion;
+--CREATE TABLE IF NOT EXISTS `champion` (
+--    `id` integer primary key auto_increment,
+--    `name` varchar(255) NOT NULL,
+--    UNIQUE KEY `name` (`name`)
+--)
+--
+drop table if exists user;
+CREATE TABLE IF NOT EXISTS `user` (
+  `id` Integer primary key AUTO_INCREMENT,
```

Comment:
keep the same style. in champion there is integer and here Integer, also autoincrement is different. Just be consistent